src/full/Agda/Interaction/Options/Base.hs

```diff
@@ -563,6 +565,7 @@ infectiveCoinfectiveOptions =
   , infectiveOption (isJust . optCubical) "--cubical/--erased-cubical"
   , infectiveOption optGuarded "--guarded"
   , infectiveOption optProp "--prop"
+  , infectiveOption optSimpUnivPoly "--simp-univ-poly"
```

Comment:
Long option names need to be spelled out, like `--simple-universe-polymorphism`. (`--prop` is an exception as it refers to the `Prop` universe which is already a cultural artefact.)

I prefer more telling names, like `--level-universe` for instance.

Currently, there is no understanding in the community what _simple universe polymorphism_ exactly should be.  But if I speculate that there were some understanding in the future, I'd guess it involves more that putting `Level` in its own universe.